client/src/views/SignupCitizen.tsx

```diff
@@ -110,132 +180,361 @@ function SignupCitizen() {
+                  <Grid item xs={12}>
+                    <label className={label}>What type on nonprofits are you interested in?</label>
+                  </Grid>
+                  <Grid item xs={12}>
+                    {makeChips()}
+                  </Grid>
+                </Grid>
+              </>
+            )}
+
+            {/* PAGE FOUR ######################################################## */}
+            {activeStep === 3 && (
+              <>
+                <Typography
+                  className={header}
+                  variant="h4"
+                  fontSize="58px"
+                  component="h1"
+                  align="left"
+                >
+                  Upload your profile icon
+                </Typography>
+                <Typography className={label} sx={{ fontWeight: 'bold' }}>
+                  Your Profile
+                </Typography>
+                <Typography>
+                  You can update this information later in the settings of your account.
+                </Typography>
+                <Grid item xs={12} sx={{ height: '50px' }} />
+                <Grid container item xs={12} alignItems="center">
+                  <Grid item xs={3}>
+                    <Avatar sx={{ bgcolor: 'gray', width: 110, height: 110 }} />
+                  </Grid>
+                  <Grid item xs={3}>
+                    <input accept="image/*" hidden id="upload-file" type="file" />
+                    <label htmlFor="upload-file">
+                      <Button
+                        className={button}
+                        component="span"
+                        color="secondary"
+                        variant="contained"
+                      >
+                        Upload
+                      </Button>
+                    </label>
+                  </Grid>
+                </Grid>
+                <Grid item xs={12} sx={{ height: '50px' }} />
+                <Typography className={label} sx={{ fontWeight: 'bold' }}>
+                  About Yourself
+                </Typography>
+                <Grid item xs={10}>
+                  <TextField multiline rows={4} fullWidth placeholder="Tell us about yourself..." />
+                </Grid>
+              </>
+            )}
+
+            {/* PAGE FIVE ######################################################## */}
+            {/* SHOWN SHEN SIGNUP DONE ######################################################## */}
+            {activeStep === 4 && (
+              <>
+                <Typography
+                  className={header}
+                  variant="h4"
+                  fontSize="58px"
+                  component="h1"
+                  align="left"
+                >
+                  Sign up Complete!
+                </Typography>
+                <Typography className={label} sx={{ fontWeight: 'bold' }}>
+                  Your name {/* USER'S NAME GOES HERE */}
```

Comment:
you can get the name and email of the user from the response
per this endpoint https://github.com/Nonprofit-Exchange-Hub/web-app/blob/f2c3e01df8c171c7ff3d0d57674467f1af073cc5/server/src/acccount-manager/account-manager.controller.ts#L47